components/FeatureHighlight/tests/unit/FeatureHighlightCustomFontTest.m

```diff
@@ -0,0 +1,64 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialFeatureHighlight.h"
+#import "../../src/private/MDCFeatureHighlightView+Private.h"
```

Comment:
In the past we just exposed the APIs that we wanted instead of importing private headers.
https://github.com/material-components/material-components-ios/blob/develop/components/Slider/tests/unit/SliderTests.m#L28

Reply:
As it's an implementation detail, I typically wouldn't expose it publicly just so I could test it.  I rank keeping the public API as small as possible more important that exposing more API to make it easier to test.

Reply:
Done.